js/card.js

```diff
@@ -0,0 +1,23 @@
+let card = document.getElementById('card');
+let firstField = document.getElementById('first');
+let lastField = document.getElementById('last');
+let titleField = document.getElementById('title');
+let websiteField = document.getElementById('website');
+let gmailField = document.getElementById('gmail');
+let githubField = document.getElementById('github');
+let phoneNumberField = document.getElementById('phone-number');
+let whatsapp = document.getElementById('whatsapp');
```

Comment:
why variables do it directly

Reply:
When you do code javascript code using the var instead of the let. I think all the browsers are not supported let. Do Some research on this.